index.html

```diff
@@ -3,314 +3,423 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 2: Javascript and the DOM</title><!-- Latest compiled and minified CSS -->
+  <title>Felony Disenfranchisement</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Fjalla+One|Exo:400,600|Ropa+Sans' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=PT+Sans+Narrow|Arimo|Dosis|Lato' rel='stylesheet' type='text/css'>
+
   <style>
     .section { padding: 1em 0; }
-    /* An "em" is equal to the height of a line of text. */
     .header {
-      background: linear-gradient(to bottom, #337ab7 0%, #135a97 100%);
-      color: white;
-      padding: 1em 0 2em;
+      text-align: left;
+    }
+    table {
+      border-collapse: collapse;
+      width: 100px;
+    }
+
+    th, td {
+      padding: 8px;
+      text-align: left;
+      border-bottom: 1px solid #ddd;
     }
     #data-table td {
-      width: 25%;
-      /* The table has four columns, so this rule makes them of equal width. */
+      width: 20%;
+      /* The table has 5 columns, so this rule makes them of equal width. */
     }
     .bar {
       height: 20px;
       /* Note that we'll set the width dynamically using Javascript. */
-      background: #337ab7;
+      background: #F03B20;
+      border-radius: 24px;
+    }
+    .btn {
+      background: none;
+      color: #FFFFB2;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+    }
+    .hidden {
+      display: none;
+    }
+    .btn:hover {
+      background-color: #FD8D3C;
+      color: #4C4C4C;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+    }
+    .btn-group .btn:active{
+      background: #FD8D3C;
+      color: #FFFFFF;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+      outline:0;
+    }
+    .btn:focus{
+      background-color: #FD8D3C;
+      color: #4C4C4C;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+      outline: 0
+    }
+    .btn.active{
+      background-color: #FD8D3C;
+      color: #4C4C4C;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+      outline:0;
+    }
+    
+
+
+    a{color: #FFFFB2;}
+    a:hover{color: #FD8D3C;}
+
+    td:nth-child(1),th:nth-child(1) { width:200px; }
+    td:nth-child(2),th:nth-child(2) { text-align: center; }
+    td:nth-child(4),th:nth-child(4) { text-align: center; }
+    
+    tr:hover td{font-weight: bold;
```

Comment:
Using `font-weight` on hover is tricky, because it can increase the amount of space the text takes up. You're right that the hover effect should increase the contrast (it's a mistake to decrease the contrast of something the user has expressed interest in by hovering their cursor), and setting the text color to pure white does do that. You might also darken the background color to make up for not having the increased font weight.